lib/home/surveys/survey_questions.dart

```diff
@@ -133,27 +185,24 @@ class RadioQuestion extends MultipleChoice {
   }
 }
 
-/// {@template views.survey.CheckboxQuestion}
+/// {@template CheckboxQuestion}
 /// A multiple-choice question with checkboxes next to each item,
 /// so you can select one or more.
 /// {@endtemplate}
 ///
 /// The answer data is a list of [bool]s: each item is `true` or `false`
 /// depending on whether the checkbox is selected.
 class CheckboxQuestion extends MultipleChoice {
-  /// {@macro views.survey.CheckboxQuestion}
+  /// {@macro CheckboxQuestion}
   const CheckboxQuestion(
     super.description, {
-    required super.choices,
+    super.choices,
     super.optional = false,
     super.canType = false,
   });
 
   @override
-  (List<bool>, String?) get initial =>
-      (List.filled(choices.length + (canType ? 1 : 0), false), null);
```

Comment:
Instead of storing the initial answer value in the question class, I thought it made sense to be moved to [`SurveyRecord`](https://github.com/theheartcenter-one/thc/blob/9986560224a519c2b4ff86bb6eeaaea0db632f5c/lib/home/surveys/take_survey/survey_field.dart#L50).